staff/maryam-malek/pre-course/bingo/main.js

```diff
@@ -0,0 +1,125 @@
+
+//Sepáralo todo en funciones, englobado en una funcion global llamada bingo(), tal que:
+
+//-Function! => Generar Numero Random Bombo
+
+//-Function! => Nuevo turno (Match carton[i] === randomNum)
+
+//-Function! => Preguntar Nuevo Turno
+
+function bingoGame(){
+	var playerName = window.prompt('Hi, could you tell me your name?', 'Ms. Malek')
+
+	var bingoCard = [
+    { number: 12, matched: false },
+    { number: 3, matched: false },
+    { number: 7, matched: false },
+    { number: 13, matched: false },
+    { number: 5, matched: false },
+    //next line
+    { number: 6, matched: false },
+    { number: 2, matched: false },
+    { number: 8, matched: false },
+    { number: 11, matched: false },
+    { number: 4, matched: false }
+    ];
+
+	console.log('Hi ' + playerName + '! This is your scorecard' + '\n' + bingoCard[0].number + '  ' + bingoCard[1].number+ '  ' + bingoCard[2].number +'  ' + bingoCard[3].number +'  ' +  bingoCard[4].number + '\n' + bingoCard[5].number + '  ' + bingoCard[6].number + '  ' + bingoCard[7].number + '  ' + bingoCard[8].number +'  ' +  bingoCard[9].number);
+    askTurn();
+
+	function generateNum(){
+		var num = Math.floor((Math.random() * 15 ) + 1);
+        return num;
+
+	}
+
+	function newTurn(){
+        var numTurn = generateNum();
+        //console.log('numTurn:'+numTurn)
+        for(var y = 0; y< bingoCard.length; y++){
+            if(bingoCard[y].number === numTurn){
+                //console.log('bingoCard number'+bingoCard[y].number)
+                bingoCard[y].number = 'X';
+                bingoCard[y].matched = true;
+                console.log(numTurn + '!!' + '\n' + "You have been lucky! You had this number in your scoreboard. Let's see how are you doing..." + '\n' + bingoCard[0].number + '  ' + bingoCard[1].number+ '  ' + bingoCard[2].number +'  ' + bingoCard[3].number +'  ' +  bingoCard[4].number + '\n' + bingoCard[5].number + '  ' + bingoCard[6].number + '  ' + bingoCard[7].number + '  ' + bingoCard[8].number +'  ' +  bingoCard[9].number)
+                askTurn();
+            }else{
+                console.log(numTurn)
+                //console.log('i: ' + y)
+                //console.log('bingoCard num'+bingoCard[y].number)
+                //console.log(numTurn + '!!' + '\n' + "In your scoreboard there wasn't this number.")
+                
+            }
+        }
+
+	}
+
+	function askTurn(){
+        var xCounter = 0;
+        for(var i = 0; i< bingoCard.length; i++){
+            if(bingoCard[i].matched === false){
+                var confirmation = confirm('Do you want to continue playing?')
+                //var tx;
+                if(confirmation === true){
```

Comment:
that type of logic operations is a bad practice, just use instead ```if(confirmation){```

Reply:
why is it bad practice to use the ===true instead of if(confirmation)?

Reply:
is redundant, the variable ```confirmation``` is already a boolean